Amazon.IonDotnet/Internals/Text/IonTextWriter.cs

```diff
@@ -225,13 +260,13 @@ public override void WriteString(string value)
 
                 // CloseValue sets followingLongString = false so we must overwrite
                 this.followingLongString = true;
-                return;
             }
-
-            // double-quoted
-            this.textWriter.WriteString(value);
-
-            this.CloseValue();
+            else
+            {
+                // double-quoted
+                this.textWriter.WriteString(value);
+                this.CloseValue();
+            }
```

Comment:
Don't we want this branch to be triggered whenever JsonDowngrade is enabled? Maybe the easiest way is to set LongStringThreshold to the maximum integer value when enabling JsonDowngrade mode.

Reply:
By default it's set to MaxValue.